webdriver/tests/bidi/browser/create_user_context/invalid.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+import webdriver.bidi.error as error
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.mark.parametrize("value", [42, "foo", {}])
```

Comment:
An empty array is also invalid value
```suggestion
@pytest.mark.parametrize("value", [42, "foo", {}, []])
```